mapr/metadata.csv

```diff
@@ -0,0 +1,158 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+mapr.metrics.submitted,gauge,,,,Number of metrics submitted every check run.,0,mapr,
+mapr.drill.allocator_root_used,gauge,10,byte,,The amount of memory used in bytes by the internal memory allocator.,0,mapr,
+mapr.drill.allocator_root_peak,gauge,10,byte,,The peak amount of memory used in bytes by the internal memory allocator.,0,mapr,
+mapr.drill.blocked_count,gauge,10,thread,,The number of threads that are blocked because they are waiting for a monitor lock.,0,mapr,
+mapr.drill.count,gauge,10,thread,,The number of live threads (including both daemon and non-daemon threads).,0,mapr,
+mapr.drill.fd_usage,gauge,10,,,The ratio of used to total file descriptors.,0,mapr,
+mapr.drill.fragments_running,gauge,10,byte,,The number of query fragments currently running in the drillbit.,0,mapr,
+mapr.drill.heap_used,gauge,10,byte,,The amount of heap memory used in bytes by the JVM.,0,mapr,
+mapr.drill.non_heap_used,gauge,10,byte,,The amount of non-heap memory used in bytes by the JVM.,0,mapr,
+mapr.drill.queries_completed,count,10,byte,,"The number of completed, canceled or failed queries for which this drillbit is the foreman.",0,mapr,
+mapr.drill.queries_running,gauge,10,byte,,The number of running queries for which this drillbit is the foreman.,0,mapr,
```

Comment:
Yeah I hesitated about this one, both `count` and `gauge` are sensical here.

Either we keep it a gauge and when looking at a graph, the values will always be "the number of queries that are currently running at that moment"

Or we make it a count, and in a graph each point will be "the total number of queries that have been in a running state between this timestamp and the next timestamp of the graph.

As said both are sensical but I decided to make it a gauge because I feel like the number of simulataneous queries at any moment would be more valuable than the total number of queries that have run. Also for getting the total number of queries that have run, the metric `mapr.drill.queries_completed` would be a good approximation for a graph with a large timewindow.

WDYT?